modules/battery_manager/README.md

```diff
@@ -0,0 +1,58 @@
+# Battery Manager
```

Comment:
Can you run the paragraphs in this through `fmt` to ensure the line lengths are consistent?

Reply:
Sure, I don't believe I've ever done that. What's an example of the command?

Reply:
To format `modules/battery_manager/README.md` and save the output into a new file `modules/battery_manager/formatted_readme`.
`fmt < modules/battery_manager/README.md > modules/battery_manager/formatted_readme`

Do this and then make sure the output makes sense. Then rename the new file and delete the old README.md